lib/billing-events-core/src/lib.rs

```diff
@@ -1,3 +1,30 @@
+//! This create provides centralized logic for working with the billing events NATS Jetstream stream.
+
+#![warn(
```

Comment:
Not for this PR, but it looks like we might be able to standardize this (hopefully across the workspace) in Cargo.toml now if we want: [[lints]](https://doc.rust-lang.org/stable/cargo/reference/manifest.html#the-lints-section) was stabilized sometime in the past year.

Reply:
Yes! We've been talking about this for awhile. That's the goal! Probably should take some time do it.

The bigger problem isn't the act itself, but ensuring the old parts of the codebase adhere to it.